src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/ApiSteps.java

```diff
@@ -231,4 +258,13 @@ public void sendGetRequestToTheLocation() {
         testContext.getHttpContext().setHttpResponseStatusCode(response.getStatusCodeValue());
         testContext.getHttpContext().setRawResponseString(response.getBody());
     }
+
+
+    @And("^the question round issued event is in the events table$")
+    public void theQuestionRoundIssuedEventIsInTheEventsTable() {
+        OnlineHearing onlineHearing = testContext.getScenarioContext().getCurrentOnlineHearing();
+        List<SessionEvent> sessionEvents = sessionEventRepository.findAllByOnlineHearing(onlineHearing);
+
+        assertTrue(!sessionEvents.isEmpty());
+    }
```

Comment:
This only tests that an event for this online hearing is in the queue, not the that it's of the correct event type